Examples/Python/ellipsoid.py

```diff
@@ -283,9 +297,17 @@ def Run_Pipeline(args):
     # Run multiscale optimization unless single scale is specified
     if not args.use_single_scale:
         parameter_dictionary["use_shape_statistics_after"] = 32
-    # Execute the optimization function
+
```

Comment:
I made this change, I am still concerned this will be confusing for users because it is less readable. But it did greatly reduce the repetition so I added it